lib/sync-settings.coffee

```diff
@@ -16,11 +14,25 @@ module.exports =
   activate: ->
     GitHubApi ?= require 'github'
     PackageManager ?= require './package-manager'
-    atom.commands.add 'atom-workspace', "sync-settings:backup", => @backup()
-    atom.commands.add 'atom-workspace', "sync-settings:restore", => @restore()
-    atom.commands.add 'atom-workspace', "sync-settings:view-backup", => @viewBackup()
+
+    Tracker ?= require './tracker'
+
+    @tracker = new Tracker('sync-settings._analyticsUserId')
+
+    @tracker.trackActivate()
+
+    atom.commands.add 'atom-workspace', "sync-settings:backup", =>
+      @tracker.track 'Backup'
+      @backup()
```

Comment:
Should backing up and tracking be reversed, in case tracking throws an error for what reason?